Amasty_Gdpr/templates/checkboxNewsletter.phtml

```diff
@@ -40,12 +40,12 @@
         >
             <use xlink:href="<?= $block->escapeUrl($block->getViewFileUrl('/images/icons-sprite.svg#unchecked')) ?>"></use>
         </svg>
-        
+
         <label
             for="amgdpr_agree_newsletter"
             class="checkbox__label"
         >
-            <?= $block->getConsentText() ?>
+            <?= $block->escapeHtml($block->getConsentText($consent), ['a']) ?>
```

Comment:
good to add to affects in changelog: 
`Affects: amasty gdpr elements and pages (newsletter, agreements, ettings, etc). Can limit html output there`

Reply:
I'm not sure if you are telling about just pasting this text to changelog file  ? If so, done.